pynetdicom/presentation.py

```diff
@@ -461,8 +461,7 @@ def __str__(self) -> str:
         s.append("Transfer Syntax(es):")
         for syntax in self.transfer_syntax:
             s.append(f"    ={syntax.name}")
-        else:
-            s.append("    (none)")
+        s.append("    (none)")
```

Comment:
Hmm, this is actually a bug, it should only add `(none)` if there's no transfer syntaxes in the context.

Reply:
ie it should be
```python

if not self.transfer_syntax:
    s.append("    (none)")
else:
    s.extend([f"    ={syntax.name}" for syntax in self.transfer_syntax])
```

Reply:
Ah, right. I'll fix that.

No need to change the iterable into a list though, this should be enough:
```python
    s.extend(f"    ={syntax.name}" for syntax in self.transfer_syntax)
```
Indeed, `list.extend()` accepts an iterable:
https://docs.python.org/3/tutorial/datastructures.html#more-on-lists
> list.**extend**(_iterable_)
> 
> Extend the list by appending all the items from the iterable. Equivalent to `a[len(a):] = iterable`.

Reply:
Oh, neat, didn't know I could do that